mmv1/templates/terraform/examples/parallelstore_instance_basic.tf.erb

```diff
@@ -0,0 +1,40 @@
+resource "google_parallelstore_instance" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Would it be possible to update this example to include the `reserved_ip_range` field?